back/engines/commercial/insights/spec/lib/finders/insights/inputs_finder_spec.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Insights::InputsFinder do
+  describe '#execute' do
+    let(:view) { create(:view) }
+    let!(:inputs) { create_list(:idea, 5, project: view.scope) }
+
+    context 'without params' do
+      let(:finder) { described_class.new(view) }
+
+      it 'returns all inputs' do
+        expect(finder.execute).to match(inputs)
+      end
+    end
+
+    context 'when view scope has not input' do
```

Comment:
Very much nitpicking
```suggestion
    context 'when view scope has no input' do
```